wonnx/tests/reduce.rs

```diff
@@ -223,6 +235,17 @@ fn reduce() {
         &[3, 2],
     );
 
+    // ONNX test case: do_not_keepdims with ArgMax
```

Comment:
Can you also enable the corresponding test case in the ONNX backend test (Python scripts)?

Reply:
Will do!